x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.test.tsx

```diff
@@ -366,13 +366,6 @@ describe('Body', () => {
           }).type,
         })
       );
-      expect(mockDispatch).toHaveBeenNthCalledWith(
```

Comment:
The business logic of automatic pinning of events with a note has been moved to the redux middleware.